software/solution-strategy.qmd

```diff
@@ -7,36 +7,10 @@ order: 4
 
 This document describes what overall strategy we have decided on to
 solve our problem, as outlined in the [Introduction
-section](introduction.qmd). Specifically, what main technologies and
-what architectural designs we will use. Any solution we decide must
-adhere to the [Principles](principles.qmd)
+section](introduction.qmd). Any solution we decide must
+adhere to the [Guiding principles](introduction.qmd#guiding-principles)
```

Comment:
```suggestion
adhere to the [Guiding principles](principles.qmd)
```

Reply:
Ah! I forgot to change this. Nice catch!